### PR TITLE
Fix bound function length

### PR DESCRIPTION
**Pull request by dbatyai**
JerryScript-DCO-1.0-Signed-off-by: Dániel Bátyai dbatyai.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by galpeter** [comment]
lgtm

File: jerry-core/ecma/builtin-objects/ecma-builtin-function-prototype.cpp [hunk 1]
```diff
@@ -279,10 +279,36 @@ ecma_builtin_function_prototype_object_bind (ecma_value_t this_arg, /**< this ar
      * See also: ecma_object_get_class_name
      */
 
-    /* 17. */
+    /* 16. */
     ecma_number_t *length_p = ecma_alloc_number ();
-    *length_p = 0;
+    *length_p = ECMA_NUMBER_ZERO;
```
Comment (ruben-ayrapetyan) [inline jerry-core/ecma/builtin-objects/ecma-builtin-function-prototype.cpp]: Could you, please, move the operation to `else` block of `if (ecma_object_get_class_name (this_arg_obj_p) == LIT_MAGIC_STRING_FUNCTION_UL)`?

**Comment by ruben-ayrapetyan** [comment]
After update according to https://github.com/Samsung/jerryscript/pull/547#discussion_r36515679, the changes would look good to me.

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Pull request merged**